File: Sorting/Sorting01.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>
using namespace std;

vector<int>solution(vector<int>array,vector<vector<int>> commands){
    vector<int> answer;
    
    for(int i=0,j=commands.size();i<j;i++){
        int targetIndex=commands[i][0]-1;
        int toIndex=commands[i][1]-1;
        int answerIndex=commands[i][2]-1;

        vector<int>newArray={};
        do{
            newArray.push_back(array[targetIndex]);
            targetIndex++;
        }while(targetIndex<toIndex+1);
        sort(newArray.begin(),newArray.end());
        answer.push_back(newArray[answerIndex]);
    }

    return answer;
}

File: Sorting/Sorting01.py
def solution(array, commands):
    tmp = []
    answer = []
    i = 0

    while i < len(commands):
        a = commands[i][0] - 1
        b = commands[i][1]
        c = commands[i][2] - 1

        tmp = array[a:b]
        
        tmp.sort()
        answer.append(tmp[c])
        i = i + 1

    return answer
